docs/ai/quickstarts/snippets/evaluate-ai-responses/MyTests.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.Extensions.AI;
+using Microsoft.Extensions.AI.Evaluation;
+using Microsoft.Extensions.AI.Evaluation.Quality;
+
+namespace TestAI;
+
+[TestClass]
+public sealed class MyTests
+{
+    // <SnippetPrivateMembers>
+    private static ChatConfiguration? s_chatConfiguration;
+    private static IList<ChatMessage> s_messages = [
+    new ChatMessage(
+            ChatRole.System,
+            """
+            You're an AI assistant that can answer questions related to astronomy.
+            Keep your responses concise and try to stay under 100 words.
+            Use the imperial measurement system for all measurements in your response.
+            """),
+        new ChatMessage(
+            ChatRole.User,
+            "How far is the planet Venus from Earth at its closest and furthest points?")];
+    private static ChatMessage s_response = new();
+    // </SnippetPrivateMembers>
+
+    // <SnippetInitialize>
+    [ClassInitialize]
+    public static async Task InitializeAsync(TestContext _)
+    {
+        /// Set up the <see cref="ChatConfiguration"/>,
+        // which includes the <see cref="IChatClient"/> that the
+        /// evaluator uses to communicate with the model.
+        s_chatConfiguration = GetOllamaChatConfiguration();
+
+        var chatOptions =
+            new ChatOptions
+            {
+                Temperature = 0.0f,
+                ResponseFormat = ChatResponseFormat.Text
+            };
+
+        /// Fetch the response to be evaluated
+        // and store it in a static variable.
+        ChatResponse response = await s_chatConfiguration.ChatClient.GetResponseAsync(s_messages, chatOptions);
+        s_response = response.Message;
+    }
+    // </SnippetInitialize>
+
+    // <SnippetGetChatConfig>
+    private static ChatConfiguration GetOllamaChatConfiguration()
+    {
+        /// Get a chat client for the Ollama endpoint.
+        IChatClient client =
+            new OllamaChatClient(
+                new Uri("http://localhost:11434"),
+                modelId: "phi3:mini");
```

Comment:
Just wanted to draw attention to what I mentioned [here](https://github.com/dotnet/ai-samples/blob/main/src/microsoft-extensions-ai-evaluation/api/INSTRUCTIONS.md?plain=1#L22C1-L27C48) in the API samples.

The evaluators have mainly been tested against `gpt-4o` and I am not sure how well `phi3Lmini` does at producing good scores reliably :) In your testing did this produce good results reliably?

Reply:
@peterwald We should probably add a doc comment remark similar to the comment I linked above on all the existing evaluators - I can look into this.

Reply:
@gewarren Just curious was `phi3:mini` working well in the above case?

Reply:
@shyamnamboodiripad The test passed every time I ran it (which was only a handful). I didn't dig any deeper than that.

Reply:
I see - that is good to know 👍🏾 :)